website/docs/tasks/generate-types.md

```diff
@@ -0,0 +1,76 @@
+---
+title: generate types
+---
+
+## Overview
+
+Taq `generate types` is used to generate TypeScript types from the Michelson smart contracts in your project. These generated types can then be used in your dApp to interact with the smart contracts using type safety
+
+The Michelson contracts that will be processed are those in the `/artifacts` directory. The default output directory will be a `/types` directory in the same directory as the `/artifacts` directory
+
+For each contract that is processed, two output files are created:
+
+- `<fileName>.types.ts` - This file contains a TypeScript definition for the contract, storage, and entrypoints and is what you will import into your TS dApp
+- `<fileName>".code.ts`  - This file contains the code that is used to interact with the contract, you will not need to import this file into your TS dApp
+
+Additionally, there are two helper files that are created:
+
+- `type-aliases.ts` - This file contains the mapping from Michelson types to TypeScript types, and some helper objects that can be used to cast types (`tas`), and instantiate bigMaps
```

Comment:
Should there be documentation somewhere that outlines what `tas` does and how to use it? I don't know if there is anything else out there in our docs that explains tas and how to use it

Reply:
Yes, we are missing documentation on the practical use of the generated types which will focus mainly on usage in a dApp

It's on the backlog but won't be included in this PR